resources/views/about.blade.php

```diff
@@ -1,5 +1,172 @@
+                    {{__("content.about.section_2.subtitle_1")}}
+                    <span class="font-medium leading-relaxed">{{__("content.about.section_2.subtitle_2")}}</span>
+                </p>
+            </div>
+        </div>
+        </div>
+    </section>
+    <section class="relative pt-20 pb-24 text-center">
+        <div class="px-[10%] md:px-[20%] space-y-10">
+            <div class="pb-14">
+                <h2 class="text-3xl md:text-3xl lg:text-5xl font-semibold pb-8 md:delay-300 md:duration-700 md:taos:scale-75 md:taos:opacity-0"
+                    data-taos-offset="400">{{__("content.about.section_3.title_1")}}</h2>
+                <span class="font-light leading-relaxed text-xl relative md:delay-300 md:duration-700 md:taos:scale-75 md:taos:opacity-0"
+                      data-taos-offset="400">{{__("content.about.section_3.subtitle_1")}}</span>
+            </div>
+            <div class="grid lg:grid-cols-3 my-6 md:justify-between lg:justify-center items-start gap-y-12 xl:gap-x-12">
+                <div class="flex flex-col items-center gap-y-6 leading-relaxed md:delay-300 md:duration-700 md:taos:scale-75 md:taos:opacity-0"
+                    data-taos-offset="400">
+                    <x-icons.passion/>
+                    <p class="text-lg lg:text-xl font-light text-center">
+                        <span class="font-medium leading-relaxed">{{__("content.about.section_3.content_1")}}</span>{{__("content.about.section_3.content_2")}}
+                    </p>
+                </div>
+                <div class="flex flex-col items-center gap-y-6 leading-relaxed md:delay-300 md:duration-700 md:taos:scale-75 md:taos:opacity-0"
+                    data-taos-offset="400">
+                    <x-icons.innovation/>
+                    <p class="text-lg lg:text-xl font-light text-center">
+                        <span class="font-medium leading-relaxed">{{__("content.about.section_3.content_3")}}</span>{{__("content.about.section_3.content_4")}}
+                    </p>
+                </div>
+                <div class="flex flex-col items-center gap-y-6 leading-relaxed md:delay-300 md:duration-700 md:taos:scale-75 md:taos:opacity-0"
+                    data-taos-offset="400">
+                    <x-icons.perfection/>
+                    <p class="text-lg lg:text-xl font-light text-center">
+                        <span class="font-medium leading-relaxed">{{__("content.about.section_3.content_5")}}</span>{{__("content.about.section_3.content_6")}}
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="bg-gradient-to-b from-gray-light to-white">
+        <x-waves.wave-2.top></x-waves.wave-2.top>
+        <div class="relative flex flex-col xl:flex-row text-center md:text-left px-[10%] md:px-[20%] gap-x-16 py-8 xl:py-28 z-20">
+            <div class="flex flex-col basis-1/3">
```

Comment:
`basis-1/3` class can be problematic in mobile version.

Reply:
switched to grid, right now grid cols 3 is used only on 2xl screens so it shouldn't be a problem